db/migrate/20190711112443_add_locked_flag_to_document.rb

```diff
@@ -0,0 +1,5 @@
+class AddLockedFlagToDocument < ActiveRecord::Migration[5.1]
+  def change
+    add_column :documents, :locked, :boolean, default: false, null: false
```

Comment:
Because this has a default it might be a rather slow migration to run. I've not worked with MySQL in a while but doing this on Postgres is slow and locking. Some googling led me to this: https://docs.gitlab.com/ee/development/migration_style_guide.html which seems to imply it can be a problem

Does it take long at all on your VM? I assume there you have basically a full set of data so is probably a reasonable approximation. There probably isn't actually that many document records so might be virtually pain free.

Reply:
Oops sorry, must have commented and merged at the same time 😬 I've got 278.5k documents locally and it took 7.68 seconds to run the migration.

Reply:
No worries. 

Ok that doesn't sound too bad a time, often the migrations can take longer on prod with all the db activity so you probably brace yourself that it could potentially cause downtime in the region of about 30s so you shouldn't supervise it's deploy.

Obviously now it's merged there isn't a chance to update commit with more details but could still update PR description if you agree that'd be useful.